codecov.yml

```diff
@@ -0,0 +1,10 @@
+comment:
+  # Only show diff and files changed:
+  layout: "diff, files"
```

Comment:
@aduh95 this configuration should reduce the coverage report to just the diff summary, and the files that changed.